tools/finalize.py

```diff
@@ -56,8 +45,11 @@ def ensure_config():
 
 
 if __name__ == '__main__':
-    print("Cutting release at", get_version())
+    version = get_version()
+    print("Cutting release at", version)
     ensure_config()
+    save_version(version)
     towncrier.check_changes()
     update_changelog()
-    bump_version()
+    subprocess.check_call(['git', 'commit', '-a', '-m', f'Finalize #{version}'])
+    subprocess.check_call(['git', 'tag', '-a', '-m', '', version])
```

Comment:
We probably can simplify this a bit more by using the latest `jaraco.develop` functions, but since this is only a preliminary draft, I did not look to much into it yet. May be improved in the future.

A possible approach to investigate would be:

```python
from jaraco.develop import towncrier
from jaraco.develop.finalize import finalize

version = towncrier.semver(towncrier.get_version())
pathlib.Path(".latest").unlink()  # Remove "unstable"/development version
pathlib.Path(".stable").write_text(version, encoding="utf-8")
subprocess.check_output(['git', 'add', ".stable"])
finalize()
```

Reply:
I might even say that we should provide hooks in jaraco.develop to implement these behaviors.